### DISCUSSION TITLE
Wifi and mqtt config on one GET request

### DISCUSSION THREAD

**Discussion by stigmat4j** [arendst/Tasmota#11861] [Q&A] [answered]
Hi all! 
I am developing a smart home application. I need to give the parameters of wifi and mqtt in one request. Now, after saving the wifi parameter, you need to wait for the controller to reload and redirect the page. Are there any options?

**Comment by lalo-uy** [comment, marked as answer]
http://192.168.1.9/cm?cmnd=backlog%20ssid1%20MyNet;password1%20MyPass;mqtthost%20192.168.1.200;topic%20KitchenFan

**Comment by rapi3** [comment]
you can use backlog command:
https://tasmota.github.io/docs/Commands/#backlog
https://tasmota.github.io/docs/Commands/#the-power-of-backlog
